narwhal/primary/src/primary.rs

```diff
@@ -194,7 +200,7 @@ impl Primary {
             .primary(&name)
             .expect("Our public key or worker id is not in the committee");
         let address = address
-            .replace(0, |_protocol| Some(Protocol::Ip4(Primary::INADDR_ANY)))
+            .replace(0, |_protocol| Some(Protocol::Ip4(Ipv4Addr::UNSPECIFIED)))
```

Comment:
Whats the purpose of switching to unspecified?

Reply:
The `Primary::INADDR_ANY` was a custom constant that we defined - whereas the `Ipv4Addr::UNSPECIFIED` is a standard one offered by the networking library (and behind the scenes is the exact same value). So thought that there is no reason to have our own constant for the exact same thing.